App/Screens/Auth/Login.js

```diff
@@ -80,7 +81,8 @@ class Login extends Component<Props, State> {
       const token = await authController.login(email, password)
       await authController.loadAndStoreUser(token.data)
       DeviceEventEmitter.emit('user.loggedIn')
-      this.navigate('Home')
+      const user = await UserController.get()
```

Comment:
`UserController.get()` is not asynchronous.
```suggestion
      const user = UserController.get()
```